test/datafile.test.js

```diff
@@ -520,7 +523,8 @@ describe('Reader()', function(){
             var count = 0;
             var fileStream = fs.createReadStream(__dirname + "/data/log.snappy.avro");
 
-            fileStream.pipe(DataFile.Reader())
+            reader = fileStream.pipe(DataFile.Reader());
```

Comment:
case in point - just in case, mocha provides `describe`-level context, so you can do `this.reader = ...` within an `it()` and any other `it()` in the same (or lower) `describe()` block will be able to access it